damus/Views/EditMetadataView.swift

```diff
@@ -160,9 +160,9 @@ struct EditMetadataView: View {
                     Text("NIP-05 Verification")
                 }, footer: {
                     if let parts = nip05_parts {
-                        Text(String.localizedStringWithFormat("'%@' at '%@' will be used for verification", parts.username, parts.host))
```

Comment:
Using the strings in this way does not make it localizable, so in my machine-translated Spanish tests, they were appearing as English.